acme/api/account.go

```diff
@@ -162,6 +162,7 @@ func NewAccount(w http.ResponseWriter, r *http.Request) {
 	linker.LinkAccount(ctx, acc)
 
 	w.Header().Set("Location", getAccountLocationPath(ctx, linker, acc.ID))
+	w.Header().Set("Cache-Control", "private, no-store")
```

Comment:
I don't see anything sensitive in the ACME account object. What data was of concern on this response?

Reply:
When I looked to see what it contained, I saw the `jose.JSONWebKey` which, if you go to the type definition, has a comment saying it "represents a public or private key". But after looking into the ACME protocol a bit, it seems to me that the server would be returning the public key here. So I'll remove the header here if you agree that we should.

Reply:
Removed the header here.